src/components/views/messages/MImageBody.tsx

```diff
@@ -113,14 +113,14 @@ export default class MImageBody extends React.Component<IProps, IState> {
                 return;
             }
 
-            const content = this.props.mxEvent.getContent() as IMediaEventContent;
+            const content = this.props.mxEvent.getContent<IMediaEventContent>();
             const httpUrl = this.getContentUrl();
-            const ImageView = sdk.getComponent("elements.ImageView");
-            const params: ImageViewIProps = {
+            const params: ComponentProps<typeof ImageView> = {
                 src: httpUrl,
                 name: content.body?.length > 0 ? content.body : _t('Attachment'),
                 mxEvent: this.props.mxEvent,
                 permalinkCreator: this.props.permalinkCreator,
+                onFinished: () => {},
             };
```

Comment:
```suggestion
            const params: Omit<ComponentProps<typeof ImageView>, "onFinished"> = {
                src: httpUrl,
                name: content.body?.length > 0 ? content.body : _t('Attachment'),
                mxEvent: this.props.mxEvent,
                permalinkCreator: this.props.permalinkCreator,
            };
```

the onFinished is injected by the ModalManager